packages/dds/tree/src/feature-libraries/editable-tree/editableTree.ts

```diff
@@ -135,6 +126,14 @@ export interface EditableTree extends Iterable<EditableField> {
      * when the fields are getting changed while iterating.
      */
     [Symbol.iterator](): IterableIterator<EditableField>;
+
+    /**
+     * Creates a new field at this node.
```

Comment:
What happens if another client concurently creates the same field?
I'll come back to update this comment thread when I find the answer in the implementation.

Reply:
For optional fields the semantics are LWW overwrite.
For sequence fields the semantics are "insert those nodes" where the contents ends up in order of sequenced-last to sequenced-first. In the near future we will expose a flag to allow choosing the opposite order.

Reply:
Thanks! Updated.